src/pint/models/jump.py

```diff
@@ -136,3 +135,22 @@ def get_jump_param_objects(self):
         """
         jump_obs = [getattr(self, jump) for jump in self.jumps]
         return jump_obs
+
+    def jump_params_to_flags(self, toas):
+        """Take jumps created from .par file and add appropriate flags to toa table.
+        This function was made specifically with pintk in mind for a way to properly
+        load jump flags at the same time a .par file with jumps is loaded (like how
+        jump_flags_to_params loads jumps from .tim files).
+
+        Parameters
+        ----------
+        toas: TOAs object
+            The TOAs which contain the TOA table to be modified
+        """
+        # for every jump, set appropriate flag for TOAs it jumps
+        for jump_par in self.get_jump_param_objects():
+            # find TOAs jump applies to
+            mask = jump_par.select_toa_mask(toas)
+            # apply to dictionaries
+            for dict in toas.table["flags"][mask]:
+                dict["jump"] = jump_par.index
```

Comment:
It was my understanding (at least with how adding jumps through pintk currently works), that jumps should not overlap. Right now, if you try to add a jump in pintk and it overlaps an existing jump, it won't let you add the jump.

Reply:
They could easily be set on purpose or by accident in par files, though - for example, JUMPing all data from a telescope and then JUMPing all data in a time range. In fact doesn't that come up in pintk?

Reply:
I am not sure, I will have to investigate further. I am pretty sure that overlapping jumps in the par file would overwrite one another so that only the last jump flag would be stored in the table, so perhaps I can store them as an array instead of a single entry? However, adding jumps through the GUI will not let you overlap jumps, so perhaps this behavior needs to be modified?